scripts/insecure_grpcurl_series.sh

```diff
@@ -52,11 +52,10 @@ SERIES_REQUEST='{
 
 GOGOPROTO_ROOT="$(GO111MODULE=on go list -f '{{ .Dir }}' -m github.com/gogo/protobuf)"
 
-pushd ${DIR}/../pkg/store/storepb/
```

Comment:
curious why this was not working :thinking: It's the same thing.

Reply:
yes this change is not relevant, just thought it is more simple as not everyone(including me) is familiar with pushd and popd